modern-gpus/nvidia-gpu.md

```diff
@@ -147,14 +148,14 @@ Quadro:
 #### Highest Supported OS: High Sierra(10.13.6)
 #### Initial Supported OS: Lion(10.7)
 
-Well of course Nvidia can't keep naming straight, some
+Since nVidia can't comply to any naming conventions, they decided to make some 600/700 series cards based on the older and not compatible Fermi architecture. These cards include:
```

Comment:
```suggestion
Since nVidia can't comply to any naming conventions, they decided to make some 600/700 series cards based on the older and incompatible Fermi architecture. These cards include:
```